### DISCUSSION TITLE
Incorrect balance in wallets after upgrade to beta

### DISCUSSION THREAD

**Discussion by MiniGamer21** [Chia-Network/chia-blockchain#10208] [Support]
I've just saw a message about 1.3 beta and upgraded (but i didn't understand why upgrade file name  is 1.2.12269 and not 1.3...)
I have a farmer's pc with 1 key/wallet and 3 farmer nodes.
In early versions it was 1 xch address for all 3 nodes and 1 equal balance. Now its shows 3 different "recieve address", 3 different balances (for the same wallet). i can;t see transactions from one node in another, one transaction was gone to nowhere (~0.5 xch,  but it's still very disappointing).
So, how to fix it? Remove all data (include db, wallet and client) from pc, install old client and sync from memkeys? On all nodes? Will it help?

Sorry for my English, now in my native Russian:
Друзья, помогите. Обновился до бетки (правда, почему то, на странице загрузке версия 1.2.12269, а не 1.3 - хотя по тексту говориться об 1.3).
У меня ферма из 3х компов, основной ставил просто, 2 вспомогательных - по инструкции для дополнительных фермеров. Все было замечательно, все фармилось на пул, везде все было синхронно. Но вот после обновления начался какой-то ужас - на всех 3 компах показывает разный баланс, разный адрес для получения, исходящие транзакции отнимают баланс именно у данного фермера, при этом часть транзакции на обновленных компах уходят в "никуда" (те, которые были отправлены на "старые" адреса, которые были до обновления клиента... Потерял так 0,5 xch - обидно). Про пул пишет, что "Этот участок NFT привязан к другому ключу." - причем это на всех обновленных клиентах так.
Что делать то? Как вернуть весь этот зоопарк обратно, учитывая что я и бд обновил? Полностью все грохнуть и зачистить, включая bd и wallet, восстановить все с memkey и дождаться полной синхронизации? Или есть способ попроще?

**Comment by MiniGamer21** [comment]
And another one - all 3 nodes write in pool section: "This plot NFT is assigned to a different key... you can not make changes".
All these nodes worked well with this pool on old client.

**Comment by loppefaaret** [comment]
running the beta release is not recommended for production systems.
that being said, as far as i can understand, you are running 3 machines, with full clients- all running the same key ?
and the reason it says 1.2.12269 is cause it is devbuild 269 on version 1.2.12 - this will be the 1.3 release once development is done.
if you could try and better explain exactly how the balances are different from the 3 machines, and exactly what transaction are not being shown ?

**Reply by MiniGamer21** [reply]
Hello. I fully understand what beta testing is and I am voicing the problem not to quarrel, but so that you can solve it in future releases.
Regarding your question - I can't show you the differences in the balance, because the client gives an error on the balance page. Restarting the client does not help fix the problem.
![2022-02-17_11-15-00](https://user-images.githubusercontent.com/99527291/154404836-6d87300a-e952-455e-be81-b6245299ab49.png)

Error code: Value -516379352085 of does nor fit uint128
